server/routes/temporaryabsences/checkTemporaryAbsenceController.test.ts

```diff
@@ -41,65 +33,60 @@ afterEach(() => {
 describe('GET checkTemporaryAbsence', () => {
   it('should redirect to authentication error page for non reception users', () => {
     app = appWithAllRoutes({ roles: [] })
-    return request(app).get('/prisoners/G0013AB/check-temporary-absence').expect(302).expect('Location', '/autherror')
+    return request(app).get('/prisoners/A1234AA/check-temporary-absence').expect(302).expect('Location', '/autherror')
   })
 
   it('should call service method correctly', () => {
     return request(app)
-      .get('/prisoners/G0013AB/check-temporary-absence')
+      .get('/prisoners/A1234AA/check-temporary-absence')
       .expect('Content-Type', /text\/html/)
       .expect(() => {
-        expect(temporaryAbsencesService.getTemporaryAbsence).toHaveBeenCalledWith('MDI', 'G0013AB')
+        expect(temporaryAbsencesService.getTemporaryAbsence).toHaveBeenCalledWith('MDI', 'A1234AA')
       })
   })
 
   it('should render the correct data in /check-temporary-absence page', () => {
     return request(app)
-      .get('/prisoners/G0013AB/check-temporary-absence')
+      .get('/prisoners/A1234AA/check-temporary-absence')
       .expect('Content-Type', 'text/html; charset=utf-8')
       .expect(res => {
         const $ = cheerio.load(res.text)
         expect($('h1').text()).toContain('This person will be returned to prison')
-        expect($('.data-qa-name').text()).toContain('John Doe')
-        expect($('.data-qa-dob').text()).toContain('1 January 1971')
-        expect($('.data-qa-prison-number').text()).toContain('G0013AB')
-        expect($('.data-qa-reason-for-absence').text()).toContain('Hospital appointment')
```

Comment:
we don't tend to test details and these are now incorrect as now using factory created version